extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -208,7 +209,7 @@ export class AmpImageViewer extends AMP.BaseElement {
 
   /**
    * Returns the boundaries of the image element.
-   * @return {!Element}
+   * @return {?Element}
```

Comment:
Is it nullable? Looks like it asserts.

Reply:
The intention was to remove the assertElement. Oops.

Reply:
The reasoning here is that we're technically not guaranteed that `this.image_` will be initialized until we return from layout callback. So there's the option between either returning a promise here and forcing the lightbox exit to wait for this to layout, or just to check in lightbox exit and skip the animation if no image has been laid out yet. Since the entire exit animation is dependent on the image actually loading, it is pointless to wait for this image to layout for just an exit animation.

Reply:
Cool, that makes a lot of sense